infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesDeployments.java

```diff
@@ -530,11 +530,12 @@ public void eventReceived(Action action, Event event) {
                   if (happenedAfterWatcherInitialization(podEvent)) {
                     containerEventsHandlers.forEach(h -> h.handle(podEvent));
                   }
-                } catch (ParseException | IllegalArgumentException e) {
+                } catch (ParseException e) {
                   LOG.error(
-                      "Failed to parse last timestamp of the event. Cause: {}. Event: {}",
-                      e.getMessage(),
-                      podEvent);
+                      String.format(
```

Comment:
why change to `String.format` ?

Reply:
```
2020-01-03 09:20:46,486[main]             [ERROR] [.w.i.k.n.KubernetesDeployments 12]   - Failed to parse last timestamp of the event. Cause: Message. Event: PodEvent{podName='podName', containerName='containerName', reason='reason', message='message', creationTimestamp='time', lastTimestamp='time2'}
```
vs
```
2020-01-03 09:20:46,498[main]             [ERROR] [.w.i.k.n.KubernetesDeployments 17]   - Failed to parse last timestamp of the event. Cause: Message. Event: PodEvent{podName='podName', containerName='containerName', reason='reason', message='message', creationTimestamp='time', lastTimestamp='time2'}
java.lang.Exception: Message
	at Scratch.main(scratch_4.java:10)
```
in this case, when we use (string, Exception) constructor we have also stacktrace printed.

Reply:
Ah, I didn't notice extra `e` arg. Anyway, it should be enough to just add `e` into `LOG.error` http://slf4j.org/faq.html#paramException

Reply:
@sparkoo you are right.